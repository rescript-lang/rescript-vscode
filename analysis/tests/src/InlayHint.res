let string = "ReScript"
let number = 1
let float = 1.1
let char = 'c'

let add = (x, y) => x + y

let my_sum = 3->add(1)->add(1)->add(1)->add(8)

let withAs = (~xx as yyy) => yyy + 1


@react.component
let make = (~name) => React.string(name)

let tuple = ("ReScript", "lol")

let (lang, _) = tuple

//^hin