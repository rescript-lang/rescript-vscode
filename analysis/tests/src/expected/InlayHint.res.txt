Inlay Hint src/InlayHint.res 0:6
[{
    "position": {"line": 5, "character": 15},
    "label": ": int",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}, {
    "position": {"line": 5, "character": 12},
    "label": ": int",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}, {
    "position": {"line": 3, "character": 8},
    "label": ": char",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}, {
    "position": {"line": 2, "character": 9},
    "label": ": float",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}, {
    "position": {"line": 1, "character": 10},
    "label": ": int",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}, {
    "position": {"line": 0, "character": 10},
    "label": ": string",
    "kind": 1,
    "paddingLeft": true,
    "paddingRight": false
}]

